scripts/us_census/pep/Population_Estimate_by_ASR/process_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to automate the testing for USA Population by Race preprocess script.
+"""
+
+import os
+from os import path
+import unittest
+from national_2000_2010 import national2000
+from state_1990_2000 import state1990
+from process import USCensusPEPByASR
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+test_data_folder = os.path.join(module_dir_, "test_data")
+op_data_folder = os.path.join(module_dir_, "test_output_data")
+input_path = os.path.dirname(os.path.abspath(__file__)) + os.sep + "input_data"
+
+
+class TestPreprocess(unittest.TestCase):
+    """
+    TestPreprocess is inherting unittest class
+    properties which further requried for unit testing
```

Comment:
All comments should with a period (".")

Reply:
Period (".") has been added to all the comments